### ISSUE TITLE
OH issue for Dev 04 attempt

### ISSUE THREAD

**Issue by OliveHamm14** [barrycumbie/dev-challenge-01#187]
**First thing, one thing I suggest to fix:**
- There is a typo on my code whoIsStrongBad.html (located in course-content) on line 10...
     - <h1>I know it has been in the back our minds...</h1> 
     - (it is missing "of" in the heading)
 
**For our product:**
-An idea I would use a bootstrap  layout (you can find on their website) to make everything look nice.
     -bootstrap link: https://getbootstrap.com/docs/4.0/examples/jumbotron/
-The test for our product on Nu still has some issues.
     -One thing I would do is add a language like "headlang=en"

**Comment by barrycumbie** [comment]
Story points? 

(This will work but don't we need to check the final product too?)

**Comment by OliveHamm14** [comment]
What do the story points intel? @barrycumbie 

I have been referencing iss #108 on how to write this and it states that the difficulty is worth 2 points, this process is not difficult in any way and I would rate it at a difficultly of 1 point. (is this what the story points should be like?) 
Or do you want bullet points underneath the User story? like in iss #108 .

And to check the product do you want me to use an example, like add in an image showing the spell checker works?

**Comment by barrycumbie** [comment]
> What do the story points intel? @barrycumbie 
>

*entail

Story points entail you estimating the effort and assigning a value. I'll negotiate with you if they are too small or large. 
> 
> 
> I have been referencing iss #108 on how to write this and it states that the difficulty is worth 2 points, this process is not difficult in any way and I would rate it at a difficultly of 1 point. (is this what the story points should be like?) 

Disagree. Think this one is at least 2 points. It involves a new tool and an extra step in dev process. 

> 
> Or do you want bullet points underneath the User story? like in iss #108 .
> 

Yes. Edit your first comment and put the SPs on it. I think they should be the first thing and above the user story 

> 
> 
> And to check the product do you want me to use an example, like add in an image showing the spell checker works?
> 
> 
Nah. Not an image (static) but instead instructions on how to use it, kinda like you already have 

Don't worry about our product. That is another issue. You are proposing a self-check for Devs to run on their own code. Like:
Before you push, 
1. Run through Nu
2. Run through VSC Spellcheck 

TLDR; just add story points and resubmit.

**Comment by OliveHamm14** [comment]
![created-an-issue](https://user-images.githubusercontent.com/66586011/109331674-c092cf00-7822-11eb-89ac-fc4f983fad82.jpg)

**Comment by OliveHamm14** [comment]
Added in story points @barrycumbie 

And also added in difficulty points

**Comment by barrycumbie** [comment]
😆 bart.

let's not use "difficulty"...that's what the "story points" capture. Like I said: 

> Yes. Edit your first comment and put the SPs on it. I think they should be the **first thing** and above the user story

**Comment by OliveHamm14** [comment]
@barrycumbie Ok, I will do that ASAP. And I'm glad you enjoyed the meme 😂
